getting_started.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Getting Started"
+nav_order: 2
+description: >
+    Getting Started with FalkorDB Graph Database.
+---
+
+
+# Getting Started with FalkorDB
+
+This guide will walk you through setting up FalkorDB, modeling a social network as a graph, 
+and accessing it using the [FalkorDB Python client](/clients) with [Cypher](/cypher).
+
+---
+
+## Prerequisites
+
+1. **FalkorDB Instance**: Set up FalkorDB (on-prem or cloud). [Learn More](/configuration).  
```

Comment:
Why does Learn More goes to /configuration ?

Reply:
cause it's about how run the Database, what would you suggest?

Reply:
I think learn more should lead to a page with additional explanations and demos not the configuration.

Reply:
But we don't have such page,

Reply:
then remove the link